README.md

```diff
@@ -31,7 +31,7 @@ addon | version | summary
 [web_dialog_size](web_dialog_size/) | 8.0.0.1.0 | A module that lets the user expand a dialog box to the full screen width.
 [web_dom_model_classes](web_dom_model_classes/) | 8.0.1.0.0 | Allows small UI changes with simple CSS
 [web_easy_switch_company](web_easy_switch_company/) | 8.0.1.0.0 | Multicompany - Easy Switch Company
-[web_environment_ribbon](web_environment_ribbon/) | 8.0.0.1.0 | Web Environment Ribbon
+[web_environment_ribbon](web_environment_ribbon/) | 8.0.1.0.0 | Web Environment Ribbon
```

Comment:
You have to clean your commits to remove the diff in this file, or there will be conflicts for sure.